### ISSUE TITLE
sitewide, CMS [AXE-CORE]: All page content MUST be contained by landmarks - div.veteran-banner

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6339]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

```diff
! Team affected: VSA Public Websites; Project: AMA ph1
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The div that contains the veteran portraits is not contained by a landmark - `<div class="veteran-banner">`. All page content **must** be contained by landmarks. 

Screen reader users can navigate to a section based on its HTML element or ARIA Landmark. It is best practice to contain all content excepting skip links, within distinct regions such as the header, nav, main, and footer. As this content is relatively decorative, it is seems less urgent. However, since screen reader users can't tell what is and isn't decorative, containing content within landmarks as the rule describes will prevent confusion.

```diff
! Note: This requires a page template change, and I'm not
! sure if this should be assigned to your team, CMS team, or sitewide.
```

### Why it matters

Navigating a web page is far simpler for screen reader users if the content splits between multiple high-level sections. Content outside of sections is difficult to find, and the content's purpose may be unclear.

Historically, HTML lacked some key semantic markers such as the ability to designate sections of the page as the header, navigation, main content, and footer. Using both HTML5 elements and ARIA landmarks in the same element is considered a best practice, but the future favors using native HTML5 element regions as browser support increases.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the content of the page by landmark regions.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination:  production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/` in browser
2. Run axe scan using browser tools
3. Verify that the div.veteran-banner is identified as not contained by landmarks.

## Possible Fixes 

This requires a page template change, to enclose this div.veteran-banner within div#content.

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - All page content must be contained by landmarks](https://dequeuniversity.com/rules/axe/3.4/region?application=AxeChrome)

## Screenshots or Trace Logs

![vsapublicw-ama-landmarks](https://user-images.githubusercontent.com/57469/75595640-3cfda100-5a5b-11ea-9124-5d0aa3771d92.png)

**Comment by MickinSahni** [comment]
Trevor, should this CMS item be transferred to you? @1Copenut

**Comment by 1Copenut** [comment]
@MickinSahni Yeah, I'll take it. It's going to be at least one VSP sprint (started yesterday, 4/2) until I can do anything with, possibly two.

**Comment by joshkimux** [comment]
@laflannery

**Comment by joshkimux** [comment]
Nice catch! Closing this one out ❤️
